### ISSUE TITLE
Lens for molecular visualization in SMILES notation

### ISSUE THREAD

**Issue by steffenslavetinsky** [Renumics/spotlight#274]
**Is your feature request related to a problem? Please describe.**
There is no way to easily visualize a molecular data in spotlight.

**Describe the solution you'd like**
It would be great to store molecular data as a string in SMILES notation in spotlight and visualize in using a Lens.

**Describe alternatives you've considered**
It is possible to render pictures of the molecules beforehand but this significantly increates dataset size and loading times.

**Additional context**
See the [wikipedia article on SMILES notation](https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system)
There already is a project on [molecular visualization](https://github.com/nglviewer/ngl) on github.
A dataset to experiment with can be obtained from [hugging face](https://huggingface.co/datasets/Lollitor/SMILES50)

**Comment by markus-stoll** [comment]
@Rishika631 Are you still working on this?

**Comment by Rishika631** [comment]
@markus-stoll  Yes I am still working on this. Please give me some more time for this issue.
